--- ../../../hike_v3/testbed/testbed.sh	2021-05-31 14:07:26.039667820 +0000
+++ testbed.sh	2021-05-28 14:11:30.656000000 +0000
@@ -104,56 +104,8 @@
 
 	# Load all the progs contained into prog.o and pin them into
 	# progs bpffs. We also pin all the maps on 'maps' bpffs.
-	bpftool prog loadall prog.o /sys/fs/bpf/progs type xdp 		\
-		pinmaps /sys/fs/bpf/maps
-
-	# Attach the program root hike loader (pinned) to the netdev enp6s0f0
-	# on the XDP hook.
-	bpftool net attach xdpdrv 					\
-		pinned /sys/fs/bpf/progs/hike_loader dev enp6s0f0
-
-	# Attach dummy xdp pass program to the netdev enp6s0f1 XDP hook.
-	bpftool net attach xdpdrv 					\
-		pinned /sys/fs/bpf/progs/xdp_pass dev enp6s0f1
-
-	# Jump Map configuration (used for carring out tail calls in HIKe VM)
-	# Let's populate the gen_jmp_table so that we can perform tail calls!
-
-	# Register allow_any eBPF/HIKe Program
-	# Prog ID is defined in minimal.h; we need to parse that file and
-	# use the macro value here... but I'm lazy... are YOU brave enough
-	# to do that? :-)
-
-	bpftool map update pinned /sys/fs/bpf/maps/gen_jmp_table 	\
-		key	hex 0b 00 00 00					\
-		value	pinned /sys/fs/bpf/progs/hvxdp_allow_any
-
-	# Register deny_any eBPF/HIKe Program, please see description above ;-)
-	bpftool map update pinned /sys/fs/bpf/maps/gen_jmp_table 	\
-		key	hex 0c 00 00 00					\
-		value	pinned /sys/fs/bpf/progs/hvxdp_drop_any
-
-	# Register count packet eBPF/HIKe Program, please see description above ;-)
-	bpftool map update pinned /sys/fs/bpf/maps/gen_jmp_table 	\
-		key	hex 0d 00 00 00					\
-		value	pinned /sys/fs/bpf/progs/hvxdp_count_packet
-
-	# HIKe Programs are now loaded, let's move on by loading the HIKe Chains.
-	# First of all we build the HIKe Chain program loader using the
-	# .hike.o object (which contains all the HIKe Chains defined so far).
-
-	# The HIKECC takes as 1) the HIKe Chains object file; 2) the eBPF map
-	# that contains all the HIKe Chains; 3) the path of the load script
-	# that is going to be generated.
-
-	${HIKECC} data/binaries/minimal_chain.hike.o			\
-		  /sys/fs/bpf/maps/hike_chain_map 			\
-		  data/binaries/minimal_chain.hike.load.sh
-
-	# Load HIKe Chains calling the loader script we just built :-o
-	/bin/bash data/binaries/minimal_chain.hike.load.sh
-
-	/bin/bash
+	# bpftool prog loadall prog.o /sys/fs/bpf/progs type xdp 		\
+	#	pinmaps /sys/fs/bpf/maps
 EOF
 
 ###
